### DISCUSSION TITLE
What is plot size (defaults to 32)? How to limit total plots size to a number of GB/TB ?

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#1671] [Q&A] [answered]
The help for `chia plots create` isn't very descriptive :(
Isn't something like plot_size * "number of plots" (--num) ?
But what is the unit? 32 of what?

**Comment by hoffmang9** [comment, marked as answer]
You should start here: https://github.com/Chia-Network/chia-blockchain/wiki/FAQ
And here: https://www.chia.net/2021/02/22/plotting-basics.html
